pkg/sql/distsqlrun/distinct_test.go

```diff
@@ -169,3 +169,47 @@ func TestDistinct(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkDistinct(b *testing.B) {
+	const numCols = 1
+	const numRows = 1000
+
+	ctx := context.Background()
+	evalCtx := tree.MakeTestingEvalContext()
+	defer evalCtx.Stop(ctx)
+
+	flowCtx := &FlowCtx{
+		Settings: cluster.MakeTestingClusterSettings(),
+		EvalCtx:  evalCtx,
+	}
+	spec := &DistinctSpec{
+		DistinctColumns: []uint32{0},
+	}
+	post := &PostProcessSpec{}
+
+	columnTypeInt := sqlbase.ColumnType{SemanticType: sqlbase.ColumnType_INT}
+	types := make([]sqlbase.ColumnType, numCols)
+	for i := 0; i < numCols; i++ {
+		types[i] = columnTypeInt
+	}
+	makeInputTable := func(numRows, numCols int) sqlbase.EncDatumRows {
```

Comment:
nit: while you're here, can you pull this out from `BenchmarkHashJoiner` too into `distsqlrun.utils_test.go`?

Reply:
Done.